Support time zones in CronExpression

paulhilliar opened the pull request:
Support time zones in CronExpression so that schedules can fire at a time of day that moves specific to the explicit zone not the zone of the JVM.

For example the expression "0 05 13,20 * * ?" (New York) should fire at 13:05/20:05 NY time every day, regardless of where the JVM sits.

Commit:
Support time zones in CronExpression so that schedules can fire at a time of day that moves specific to the explicit zone not the zone of the JVM.

kagkarlsson commented:
Excellent PR! Thanks for contributing!

Could you add a `cron(String, ZoneId)` method to the `Schedules` class as well?

Commit:
Support time zones in CronExpression so that schedules can fire at a time of day that moves specific to the explicit zone not the zone of the JVM.

Incorporate review feedback - add Schedules.cron(String cronPattern, ZoneId zoneId)

paulhilliar commented:
Thanks.  Added the Schedules method.

kagkarlsson commented:
Thanks! Merging!

kagkarlsson commented:
Should be available in maven central in version 6.1 soon

kagkarlsson commented:
🎉 This issue has been resolved in `v16.9.0` ([Release Notes](https://github.com/kagkarlsson/db-scheduler/releases/tag/v16.9.0))
